packages/driver/src/cypress/log.ts

```diff
@@ -613,25 +631,5 @@ function create (Cypress, cy, state, config) {
     return log
   }
 
-  logFn._logs = logs
```

Comment:
I removed this line because it doesn't do anything. And it might make typing hard later.

It was added at 2c8b9af0b642a6ad7b587d6fc330b427089ea482 for test. And changed to this style at 22f43ba49d1308f66298fc097640d5885312d525.

But it is not used anywhere currently.

Reply:
Thanks for the clean up!